### DISCUSSION TITLE
Welcome to Heyy-chat-app Discussions!

### DISCUSSION THREAD

**Discussion by IshuPrabhakar** [IshuPrabhakar/Heyy-chat-app#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by skymagic07** [comment]
Flutter application is not work SignalR and WebRTC .
Would you complete flutter application please.

**Comment by Ismael-Prieto** [comment]
Hi, could you please upgrade backend project?

**Comment by learner291** [comment]
Hi,

Do you have any blog or instructions showing how backend and frontend was developed so that we can learn?

Thanks

**Reply by shakilsheikhb2k781-cpu** [reply]
Hlw
